src/common-imports.ts

```diff
@@ -26,9 +27,11 @@ export default function registerCustomComponents(extra = {}): void {
     ...blockResourceEditorsComponents,
     ...resourceEditorComponents,
     ...extra,
-  })
+  }
+  Object.entries(Components)
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     .forEach(([name, component]: [string, any]) => {
       Vue.component(name, component)
     })
+  return Components
```

Comment:
These changes in `src/common-imports.ts` are unrelated to the bug.
They fix the return from the `registerCustomComponents()` that is used in the file below for default exports (in `src/lib.ts`)

Reply:
Ok, thanks for doing that! 🥇